README.md

```diff
@@ -132,3 +132,8 @@ Non-development URLs are also accessible using the `gds paas open` command.
 | `GITHUB_PASSWORD_STORE_DIR` | Sets the path to a [`pass`](https://www.passwordstore.org/) store where Github credentials can be found. Used in `make upload-github-oauth` |  `~/.paas-pass`|
 | `GOOGLE_PASSWORD_STORE_DIR` | Sets the path to a [`pass`](https://www.passwordstore.org/) store where Google credentials can be found. Used in `make upload-google-oauth`|  `~/.paas-pass`|
 |`ENABLE_GITHUB` | Enables Github authentication for Concourse | `true` |
+
+## terraform concourse_egress_cidrs var
+This var is used by terraform to add the cidrs of all the concourse worker nodes (previously only the current concourse worker was added) to various security groups to provide access to resources from the concourse workers. At the end of the pipeline there is an expunge-concourse job that removes the cdrs from the security groups so that the workers no longer have access to the resources.
```

Comment:
cdrs > cidrs

Reply:
The typo is corrected

Reply:
I have corrected the concourse_egress_cidrs var problem by setting it from a concourse_egress_cidrs.tfvars file.

Reply:
PLEASE DON'T MERGE THIS YET